src/Diagnostics.RuntimeHost/Controllers/DiagnosticControllerBase.cs

```diff
@@ -89,10 +94,29 @@ protected async Task<IActionResult> ListDetectors(TResource resource, string que
             {
                 return BadRequest("Search query term should be at least two characters");
             }
-            return Ok(await this.ListDetectorsInternal(cxt, queryText));
+
+            IEnumerable<DiagnosticApiResponse> listDetectorsResponse = await this.ListDetectorsInternal(cxt, queryText);
+            IEnumerable<DiagnosticApiResponse> listDetectorsTranslatedResponse = listDetectorsResponse;
+
+            try
+            {
+                if (IsLocalizationApplicable(language))
+                {
+                    listDetectorsTranslatedResponse = await _diagnosticTranslator.GetMetadataTranslations(listDetectorsResponse, language).ConfigureAwait(false);
```

Comment:
You can remove the `ConfigureAwait(false)`

Reply:
> You can remove the `ConfigureAwait(false)`
Oh Hawk I thought using ConfigureAwait(false) is a good practice to not capturing current context and avoid deadlock?

Reply:
This is a nice resource https://devblogs.microsoft.com/dotnet/configureawait-faq/#when-should-i-use-configureawaitfalse

But generally, since your not writing library code and these are all application code changes that your implementing and we know that you are already following async/await best practices when calling the Cognitive Service API then you can avoid calling `ConfigureAwait(false)`. 

Deadlock would be something to think about if we were calling some library code in which we had no idea how it was implemented (worst case it was not implemented with thread-safety in mind and broke best practices)

Reply:
Thanks Hawk! I removed it accordingly